dom/examples/drivertest/src/lib.rs

```diff
@@ -78,11 +78,11 @@ fn mutiple_event_listeners() {
 
         mox! {
             <button
-                onclick={ move |_| counter1.update(increment) }
-                onclick={ move |_| counter2.update(increment) }
+                onclick = move |_| counter1.update(increment)
+                onclick = move |_| counter2.update(increment)
             >
                 // Display the values of the counters
-                {% "counter1 = {}, counter2 = {}", &counter1_val, &counter2_val }
+                {format_args!("counter1 = {}, counter2 = {}", &counter1_val, &counter2_val)}
```

Comment:
Could you try using https://github.com/stoically/syn-rsx/issues/9#issuecomment-753609348 to avoid this part of the diff?

Reply:
Done!